android/app/src/main/java/com/textile/textilenode/TextileImageViewManager.java

```diff
@@ -34,7 +34,7 @@ public TextileImageView createViewInstance(ThemedReactContext context) {
 
     @ReactProp(name = "imageId")
     public void setImageId(TextileImageView textileImageView, String imageId) {
-        textileImageView.setImageId(imageId);
+        textileImageView.setImagePath(imageId + "/0");
```

Comment:
@sanderpick I'm going to deal with that creating of the "path" to pass to `imageFileDataForMinWidth` here in the native code. The JS-facing property is simply `imageId`, and this code appends `/0` to it. That's a safe assumption to make for now, right?

Reply:
You don't want to make it generic so it can traverse other types of files from other schemas?

Reply:
I was imagining you'd leverage `index` from the file object to create the path. Like: `<ThreadFilesInfo.target>/<ThreadFileInfo.index>`

Reply:
Oh I see. Yea we can do that. So `TextileImage` would have `target` and `index` props on it in the JS layer?

Reply:
Done.